File: python/count_words.py
def count_words(text):
	return str(len(text.split(" "))) + " words in the analyzed text"

def count_from_file(file_location):
	file = open(file_location, 'r')
	text = str(file.read())
	return count_words(text)

print count_words("This is a sentence with 7 words")
print count_from_file("paragraph.txt")

File: java/src/projects/CountWords.java
package projects;

import java.io.File;
import java.io.FileNotFoundException;
import java.util.Scanner;

public class CountWords {
	public static void main(String[] args) {
		CountWords("This is a sentence with 7 words");
		CountWords("po op");

		CountWordsFromFile("src/projects/paragraph.txt");
	}
	private static void CountWords(String inputString) {
		System.out.println(inputString.split(" ").length + " words in the analyzed text");
	}
	private static void CountWordsFromFile(String filePath) {
		 try {
			String text = new Scanner( new File(filePath) ).useDelimiter("\\A").next();
			CountWords(text);
		} catch (FileNotFoundException e) {
			e.printStackTrace();
		}
	}
}
